src/deploy.ts

```diff
@@ -141,6 +141,19 @@ class Deployer {
     }
 
     if (!currentUser) {
+      if (!this.effects.isTty) {
+        if (authError === "unauthenticated" || !apiKey) {
```

Comment:
Do we need to handle the `"forbidden"` or `null` cases of `authError` here too?

Reply:
Looks like if `authError` is `null` that implies that `apiKey` is also `null`, so that is already covered. And if `authError` is `"forbidden"`, then the `else` block displays the rejection here. It’s a big hard to follow but it looks correct. 😅